### PR TITLE
refactored all components that could be a functional component from a…

### PR DESCRIPTION
**Pull request by sherwinmina**
… class component

### CODE REVIEW

File: web/src/components/Cloud9SetupForm/Cloud9SetupForm.js [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
-    const {
-      submitting,
-      invalid,
-      status
-    } = this.props
+    const { submitting, invalid, status } = this.props;
 
     if (invalid) {
-      return "disabled"
+      return "disabled";
```
Comment (MaxWofford) [inline web/src/components/Cloud9SetupForm/Cloud9SetupForm.js]: We don't use semicolons anywhere else in the codebase

File: web/src/components/Cloud9SetupForm/Cloud9SetupForm.js [hunk 2]
```diff
@@ -1,38 +1,38 @@
-import React, { Component } from 'react'
-import { reduxForm, Field } from 'redux-form'
-import { Button, Emoji, TextField } from 'components'
-import cloud9SetupValidation from './cloud9SetupValidation'
+import React, { Component } from "react";
```
Comment (MaxWofford) [inline web/src/components/Cloud9SetupForm/Cloud9SetupForm.js]: All the import statements in the codebase use single quotes

**Comment by MaxWofford** [comment]
@sherwinmina: Just reviewed it and left comments.

To give a bit more background, right now code styling is ambiguous because we don't have a standard style throughout the codebase ( https://github.com/hackclub/monolith/issues/219). Until that's set up I'm going to be extra careful of stuff like having semicolons and changing the usage of single/double quotes because unless those changes are across the whole codebase they're adding to the inconsistency.

Can you remove the semicolon and single/double quote changes you made and keep the rest in the PR?

**Comment by sherwinmina** [comment]
Thanks Max, I will make those changes.

**Comment by MaxWofford** [comment]
@sherwinmina: Could you fix the merge conflicts?

**Comment by MaxWofford** [comment]
Ping on this. If you've got specific questions on how to do that I'm happy to answer them.

**Comment by sherwinmina** [comment]
Hey Max, Sorry for the delay I'll work on this tomorrow if you are at Bradfield, I could definitely use the help.

**Comment by MaxWofford** [comment]
Moved to https://github.com/hackclub/monolith/pull/252 so a few changes could be made

**Pull request closed without merging**